civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5024,8 +5024,11 @@ function _civihr_employee_portal_should_do_onboarding($account) {
 
   $canViewOnboardingForm = node_access('view', $onboardingWebform, $account);
   $submissions = WebformHelper::getUserSubmissionsByTitle($account, $formName);
+  $user_wants_to_test = isset($_GET['testing']);
```

Comment:
I choose "testing" because the context is defined by the URL, from my point of view. "test_onboarding_wizard" seems repetitive to me, but I'm confident on your criteria so I will change it to the latter option.

Reply:
I will change variables to camelCase to follow the way that the function was written, although snake_case provides a better readability in my opinion.

Reply:
@powdevel now with your explanation, "testing" does sounds better and less redundant. No need to change that